internal/kafka/internal/services/clusters_test.go

```diff
@@ -940,7 +976,7 @@ func Test_ScaleDownComputeNodes(t *testing.T) {
 	}
 }
 
-func TestClusterService_ListGroupByProviderAndRegion(t *testing.T) {
+func Test_ClusterService_ListGroupByProviderAndRegion(t *testing.T) {
```

Comment:
here `ClusterService` should be `clusterService` as what is being tested is the implementation not the interface itself